dali/test/operators/passthrough.h

```diff
@@ -35,9 +35,15 @@ class PassthroughOp : public Operator<Backend> {
     return false;
   }
 
+
   void RunImpl(Workspace &ws) override {
     ws.Output<Backend>(0).ShareData(ws.Input<Backend>(0));
+    ws.SetOperatorTrace("test_trace", make_string("test_value", iteration_id++));
```

Comment:
One thing I am not very fond of is the fact, that this is operator intended to be used in some test and exhibit specific behaviour. I won't mind creating another one that is used to test traces, but I am not sure if we should pile all functionality that we want to test into it. It's not a PassthroughAndSetTraceOp, and I don't want it to become PassthroughAndSetTraceAndDoSomeOtherThingOrTwoOp.

Reply:
Added another operator. 100 extra lines of code :D